common-lib/src/main/scala/com/gu/mediaservice/model/leases/MediaLease.scala

```diff
@@ -10,6 +10,12 @@ sealed trait MediaLeaseType { def name: String }
 object MediaLeaseType {
   implicit val reads: Reads[MediaLeaseType] = {
     JsPath.read[String].map {
+      case "AllowUseLease"          => AllowUseLease
```

Comment:
These changes feel out of scope for this PR as it changes how we serialise/deserialise. Can it be moved into it's own PR?

Reply:
you are correct :)  thanks
But i will need to add them in next PR

Reply:
done